code/modules/mafia/roles.dm

```diff
@@ -467,7 +467,6 @@
 /datum/mafia_role/lawyer/proc/release(datum/mafia_controller/game)
 	SIGNAL_HANDLER
 
-	. = ..()
```

Comment:
Parent call in proc definition. You don't want to know when that has actual effects, but this isn't one of those cases. This calls nothing.

Reply:
I want to know when this has actual effects 🥺

Reply:
Trust me, you don't and you'll never encounter it unless I open that Pandora's Box for you.

I'll give you one truly cursed hint: `_dm_interface = 129`

Reply:
🤐<-- PUN TO INDICATE I HAVE LEARNED WHAT IT IS

Reply:
MSO also reminded me that they're necessary in proc redefinitions at the same level, but that wasn't what I was referring to.